data/discovery/field/data-sharing-method.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: text
```

Comment:
`string` vs `text`. Is it better to restrict this field as a non-formatted string?